ui/job-view/headerbars/SecondaryNavBar.jsx

```diff
@@ -236,6 +233,8 @@ class SecondaryNavBar extends React.PureComponent {
             {serverChanged && (
               <span
                 className="btn btn-sm btn-view-nav nav-menu-btn"
+                role="button"
+                tabIndex="0"
```

Comment:
We are transitioning towards using react-strap in our code base and I think the <Button> component will get this lint rule passing without having to add the type attribute. It's worth trying that - you could post a screen shot to see if it makes a big difference in appearance.

Reply:
I don't get this, by type attribute do you mean `role="button"`, `tabIndex="0"` or both?

Currently, this component doesn't make any difference in appearance with or without the `role` and `tabIndex` attribute.

### Without the role and tabIndex attribute
![Without the role and tabIndex attribute](https://user-images.githubusercontent.com/36057474/66152822-3422b380-e612-11e9-83ca-8d0dfb0b5129.png)

### With the role and tabIndex attribute
![With the role and tabIndex attribute](https://user-images.githubusercontent.com/36057474/66152824-34bb4a00-e612-11e9-9860-2aae5f5723f0.png)

Reply:
Sorry, I meant to say that we use a library called `react-strap` and there is a `Button` component that if used would provide the role attribute (and maybe a few others that were manually added to the button element).

Reply:
So you can try using that Button react component instead of the button html element and see if the tests still pass and the appearance is about the same.

Reply:
Okay, I understand now.

Reply:
With the `Button` component, the appearance is the same. My only observation is the `fucos` pseudoclass styling difference of the button element. 

### With Button component

![With Button component](https://user-images.githubusercontent.com/36057474/66155958-a4343800-e618-11e9-920b-d6dafd1f7932.png)

Reply:
Ok, that is probably fine. We can see what Cam thinks. You can add this change and the other change to a separate commit and update the pr :)

Reply:
okay, thanks. :blush: